package.json

```diff
@@ -62,7 +62,7 @@
     "messages"
   ],
   "repository": "eslint/eslint",
-  "funding": "https://opencollective.com/eslint",
+  "funding": "https://eslint.org/donate",
```

Comment:
should we update this as well or its fine to have open collective?

Reply:
Yes, good catch!